express/services/users.js

```diff
@@ -1,11 +1,15 @@
 const models = require('../../sequelize/models');
 
-function getAll({ filterField, filterValue, sortField = 'id', sortDirection = 'ASC' }) {
+function getAll({
+    filterField, filterValue, sortField = 'id', sortDirection = 'ASC', pageNumber = 1, instancesNumber = 5
+}) {
     const findAllOptions = {
         ...filterField && filterValue && { where: { [filterField]: filterValue } },
         order: [
             [sortField, sortDirection]
-        ]
+        ],
+        offset: instancesNumber * (pageNumber - 1),
+        limit: instancesNumber
```

Comment:
Since this logic is the same for all models services maybe it's time to move it somewhere to one place?